sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/TestBigQuery.java

```diff
@@ -211,7 +211,7 @@ public TableReference tableReference() {
   public TableDataInsertAllResponse insertRows(Schema rowSchema, Row... rows) throws IOException {
     List<Rows> bqRows =
         Arrays.stream(rows)
-            .map(row -> new Rows().setJson(BigQueryUtils.toTableRow(row)))
```

Comment:
Could you also add an overload, `BigQueryUtils.toTableRow(row)`, that preserves the existing API like you did with `BigQueryUtils.toTableRow()`? Then you wouldn't have to change calls like this one, and we preserve the existing API which is technically public (although practically no one should be relying on this).

Reply:
@TheNeuralBit Sure, I've added them.